android/java/org/chromium/chrome/browser/vpn/BraveVpnCalloutDialogFragment.java

```diff
@@ -0,0 +1,100 @@
+
+import android.content.DialogInterface;
+import android.content.res.Configuration;
+import android.os.Bundle;
+import android.util.DisplayMetrics;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.WindowManager;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+
+import androidx.fragment.app.DialogFragment;
+
+import org.chromium.chrome.R;
+import org.chromium.chrome.browser.util.ConfigurationUtils;
+import org.chromium.chrome.browser.vpn.BraveVpnPrefUtils;
+import org.chromium.chrome.browser.vpn.BraveVpnUtils;
+import org.chromium.ui.base.DeviceFormFactor;
+
+public class BraveVpnCalloutDialogFragment extends DialogFragment implements View.OnClickListener {
+    @Override
+    public void onConfigurationChanged(Configuration newConfig) {
+        super.onConfigurationChanged(newConfig);
+        setDialogParams();
+    }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+        getDialog().setOnKeyListener(new DialogInterface.OnKeyListener() {
+            @Override
+            public boolean onKey(android.content.DialogInterface dialog, int keyCode,
+                    android.view.KeyEvent event) {
+                if ((keyCode == android.view.KeyEvent.KEYCODE_BACK)) {
+                    dismiss();
+                    return true;
+                } else
+                    return false;
+            }
+        });
+        setDialogParams();
+    }
+
+    @Override
+    public View onCreateView(
+            LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.fragment_vpn_callout_dialog, container, false);
+    }
+
+    @Override
+    public void onViewCreated(View view, Bundle savedInstanceState) {
+        Button mEnableButton = view.findViewById(R.id.btn_enable);
+        mEnableButton.setOnClickListener(this);
+
+        ImageView btnClose = view.findViewById(R.id.modal_close);
+        btnClose.setOnClickListener(this);
+
+        BraveVpnPrefUtils.setBraveVpnBooleanPref(BraveVpnPrefUtils.PREF_BRAVE_VPN_CALLOUT, false);
+    }
+
+    @Override
+    public void onClick(View view) {
+        if (view.getId() == R.id.btn_enable) {
+            BraveVpnUtils.openBraveVpnPlansActivity(getActivity());
+        }
+        dismiss();
+    }
+
+    private void setDialogParams() {
+        DisplayMetrics displayMetrics = new DisplayMetrics();
+        getActivity().getWindowManager().getDefaultDisplay().getMetrics(displayMetrics);
+        int mDeviceHeight = displayMetrics.heightPixels;
+        int mDeviceWidth = displayMetrics.widthPixels;
+
+        WindowManager.LayoutParams params = getDialog().getWindow().getAttributes();
+        boolean isTablet = DeviceFormFactor.isNonMultiDisplayContextOnTablet(getActivity());
+        boolean isLandscape = ConfigurationUtils.isLandscape(getActivity());
+        if (isTablet) {
```

Comment:
maybe like in https://github.com/brave/brave-core/pull/9874/files#diff-65a8bd3ae03890980656821b506f03ad1aac7a8fc47769b9e84ce1e41d62e4e4R50 
make 
```
if (isTablet || isLandscape) {
...
} else {
...
}
```

Reply:
Now there is no the code I had the comments.

Reply:
oh, it is also in `BraveVpnDialogFragment` now :+1: